cmd/axelard/cmd/root.go

```diff
@@ -201,7 +204,12 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig params.EncodingConfig) {
 		SetGenesisAuthCmd(app.DefaultNodeHome),
 	)
 
-	server.AddCommands(rootCmd, app.DefaultNodeHome, newApp, export(encodingConfig), crisis.AddModuleInitFlags)
+	starterFlags := func(startCmd *cobra.Command) {
+		crisis.AddModuleInitFlags(startCmd)
+		startCmd.Flags().String(wasmDirFlag, "", "path to the wasm directory, default option stores it under the home/db_dir")
```

Comment:
shouldn't we want the data folder be the default? otherwise people will have to set this flag always from now on

Reply:
data folder path can be overridden via a flag as well. empty string is treated as the default (mentioned in the the help text) later in app.go where we set it to `axelard_home/db_dir/wasm`

Reply:
home is a bit confusing, prefer to make explicit `"node's home/db_dir"`  or `"axelard home/dir"`

Reply:
maybe something like "default sets it to a wasm folder inside the --db_dir data folder"?

Reply:
ya, like that

Reply:
done